website/versioned_docs/version-3.1.1/guides/markdown-features/markdown-features-diagrams.mdx

```diff
@@ -83,3 +83,4 @@ export default {
 ```
 
 See the [Mermaid config documentation](https://mermaid-js.github.io/mermaid/#/./Setup?id=configuration) and the [Mermaid config types](https://github.com/mermaid-js/mermaid/blob/master/packages/mermaid/src/config.type.ts) for the available config options.
+
```

Comment:
Is this change expected?

Don't you want to backport your docs change to this version too?

Our linter is probably not going to like it, you can run `yarn format` before commiting

Reply:
Refreshed: backported and `yarn format` done